### PR TITLE
Created requirements.txt

### PR DESCRIPTION
**Pull request by thehappydinoa**

### CODE REVIEW

**Comment by deathsec** [comment]
for installing dependencies?

**Comment by deathsec** [comment]
I suppose it does not satisfy request[socks] ? (A.K.A PySocks)

**Comment by thehappydinoa** [comment]
@deathsec I added PySocks to the dependencies list

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+stem==1.6.0
+requests==2.18.1
+setuptools==36.2.7
```
Comment (deathsec) [inline requirements.txt]: I think this need to be removed

Comment (deathsec) [reply requirements.txt]: setuptools need to be installed through a package manager

**Comment by deathsec** [comment]
did this work well for you?

**Comment by thehappydinoa** [comment]
@deathsec This worked well for me...

**Comment by deathsec** [comment]
I will get it tested later and hopefully get it merged soon , thank you for your contribution!

**Comment by thehappydinoa** [comment]
@deathsec Sorry about that I thought it was self explanatory.

**Comment by deathsec** [comment]
@thehappydinoa please create a new pull request on the v1.x.x branch of this repo , because the master branch is currently the stable version and i am planning to make some upgrades to v1.x.x and then release it later.

**Pull request closed without merging**